docs/getting-started.md

```diff
@@ -110,7 +110,7 @@ For more information on what to do next, we recommend the following sections:
 <div class="card">
   <a href="/docs/basic-features/built-in-css-support.md">
     <b>CSS Support:</b>
-    <small>Use the built-in CSS support to add custom styles to your app.</small>
+    <small>Learn more about how to use the built-in CSS support to add custom styles to your app.</small>
```

Comment:
```suggestion
    <small>Built-in CSS support to add custom styles to your app.</small>
```

Reply:
"Learn more about how to use" could make this a bit long
<img width="843" alt="CleanShot 2022-02-12 at 20 56 24@2x" src="https://user-images.githubusercontent.com/9113740/153736322-135e3259-c8c4-4aec-9ed3-82d005e69176.png">